src/main/java/com/code_intelligence/jazzer/Jazzer.java

```diff
@@ -263,6 +264,36 @@ private static Path javaBinary() {
   }
 
   private static Stream<String> javaBinaryArgs() {
+    if (isAndroid()) {
+      // Add Android specific args
+      String jazzer_agent_path = System.getProperty("jazzer.agent_path");
+      String bootclass_class_overrides =
+          System.getProperty("jazzer.android_bootpath_classes_overrides");
+
+      String srcFile = "com/code_intelligence/jazzer/android/jazzer_bootstrap_android.jar";
+      String outputPath = "/data/local/tmp/jazzer_bootstrap_android.jar";
+
+      try {
+        ZipUtils.extractFile(jazzer_agent_path, srcFile, outputPath);
+      } catch (IOException ioe) {
+        Log.error(ioe);
+        exit(1);
+      }
+
+      String nativeAgentOptions = "injectJars=" + outputPath;
+      if (bootclass_class_overrides != null && bootclass_class_overrides != "") {
+        nativeAgentOptions += ",bootstrapClassOverrides=" + bootclass_class_overrides;
```

Comment:
I assume the comma here is intentional, but you could consider Strings.join-ing them with the comma set as delimiter to make it more obvious.